### ISSUE TITLE
Techies - inconsistent and misleading "Proximity Mines" radius

### ISSUE THREAD

**Issue by Nikita0x** [ValveSoftware/Dota2-Gameplay#22567]
### Hero and spell

Techies - "Proximity Mine"

### Description

Actual radius for the mine to be triggered is bigger than the tooltip is saying ( Actual is 500, while tooltip says and shows 400). 

### Screenshots

![bug1](https://github.com/user-attachments/assets/35efcec5-e1a2-4a15-a5d4-13f8c9a129df)
![bug2](https://github.com/user-attachments/assets/6b548787-c3be-4d0e-9beb-30f2280cc546)
![bug3](https://github.com/user-attachments/assets/94b332a7-6038-4e0b-bea0-99e1d341bfb0)

**Comment by LokeLok** [comment]
The tooltip has the correct info. The green icon indicates cast range and the blue icon indicates radius of abilities.

**Comment by Nikita0x** [comment]
> The tooltip has the correct info. The green icon indicates cast range and the blue icon indicates radius of abilities.

Okay, thanks for clarifying information about tooltip.
But on the first 2 pictures you can clearly see the radius which is shown when you plan to place the mine, and on the second - the radius when the mine is placed. It is bigger, and it is misleading

**Comment by LokeLok** [comment]
Yes, you are correct that the blue radius indicator is too small. Not a tooltip issue, it's rather an issue with the radius indicator on the map.

**Comment by sPIDERsdasdasdsaf** [comment]
The blue indicator on the ground indicates the radius in which you can't place another mine near that mine. This is much more useful than showing the explosion radius.

https://github.com/user-attachments/assets/38192576-43bc-4779-a9ac-c6a534e1f48e
